ha/remote_execution/remote_executor.py

```diff
@@ -30,6 +30,6 @@ def __init__(self, hostname, port):
         self._port = port
 
     @abc.abstractmethod
-    def execute(self, command):
+    def execute(self, command, secret=None):
```

Comment:
Why this change?

Reply:
Akash keep defination same on both class
```
def execute(self, command: str, secret=None) -> None:
def execute(self, command, secret=None):
```
Keep it
def execute(self, command: str, secret: str=None):

Reply:
This is RemoteExecutor parent class's execute method. For overriding, function must have same defination.

Reply:
> Akash keep defination same on both class
> 
> ```
> def execute(self, command: str, secret=None) -> None:
> def execute(self, command, secret=None):
> ```
> 
> Keep it
> def execute(self, command: str, secret: str=None):

Function defination changed to : 
def execute(self, command: str, secret: str=None):

Reply:
Why we need this change @ajay-paratmandali ?

Reply:
We are overriding `execute` method from parent class 
In overriding method should have same parameter.

If we keep different in parent and child class then we will get error
There is also warning from codacy
```
Parameters differ from overridden 'execute' method
```
as Execute method parameter different in parent and child class